README.md

```diff
@@ -1 +1 @@
-# CodeReview
\ No newline at end of file
+# microverser-htmlcss
```

Comment:
I think it will be good for you to add a detailed readme file to your project, This will help anyone visiting your repo to know more about the project as well as all installation guide author(s).